stubs/pexpect/pexpect/utils.pyi

```diff
@@ -0,0 +1,10 @@
+from _typeshed import Incomplete
+
+InterruptedError: Incomplete
```

Comment:
I think we can just remove this for now. It's just a Python 2 compatibility stub in the implementation, which I'm not sure we need. Alternatively you could import it from builtins:

```python
from builtins import InterruptedError as InterruptedError
```